test/e2e/specs/editor/blocks/quote.spec.js

```diff
@@ -3,30 +3,189 @@
-<p>Quote content</p>
-<!-- /wp:paragraph --></blockquote>
-<!-- /wp:quote -->`
-		);
+		test( 'and renders only one paragraph for the cite, if the quote is void', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.insertBlock( { name: 'core/quote' } );
+			await page.keyboard.press( 'ArrowRight' );
+			await page.keyboard.type( 'cite' );
+			// Unwrap the block.
+			await editor.transformBlockTo( '*' );
+			expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+		} );
+
+		test( 'and renders a void paragraph if both the cite and quote are void', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.insertBlock( { name: 'core/quote' } );
+			// Select the quote
+			await page.keyboard.press( 'ArrowRight' );
+			// Unwrap the block.
+			await editor.transformBlockTo( '*' );
+			expect( await editor.getEditedPostContent() ).toBe( '' );
+		} );
+	} );
+
+	test( 'can be created by converting a heading', async ( {
+		editor,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/heading' } );
+		await page.keyboard.type( 'test' );
+		await editor.transformBlockTo( 'core/quote' );
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'can be converted to a pullquote', async ( { editor, page } ) => {
+		await editor.insertBlock( { name: 'core/quote' } );
+		await page.keyboard.type( 'one' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( 'two' );
+		await page.keyboard.press( 'ArrowRight' );
+		await page.keyboard.type( 'cite' );
+		await editor.transformBlockTo( 'core/pullquote' );
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'can be split at the end', async ( { editor, page } ) => {
+		await editor.insertBlock( { name: 'core/quote' } );
+		await page.keyboard.type( '1' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.press( 'Enter' );
+		// Expect empty paragraph outside quote block.
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+		await page.keyboard.press( 'Backspace' );
+		await page.keyboard.type( '2' );
+		// Expect the paragraph to be merged into the quote block.
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'can be unwrapped on Backspace', async ( { editor, page } ) => {
+		await editor.insertBlock( { name: 'core/quote' } );
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+		await page.keyboard.press( 'Backspace' );
+		expect( await editor.getEditedPostContent() ).toBe( '' );
+	} );
+
+	test( 'can be unwrapped with content on Backspace', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await editor.insertBlock( { name: 'core/quote' } );
+		await page.keyboard.type( '1' );
+		await page.keyboard.press( 'ArrowRight' );
+		await page.keyboard.type( '2' );
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+		await pageUtils.pressKeyTimes( 'ArrowLeft', 4 );
```

Comment:
Originally,[ the arrow-left was used for keystrokes](https://github.com/WordPress/gutenberg/blob/3245d1e0518d6a2fa565266981fe0fab8a984a8e/packages/e2e-tests/specs/editor/blocks/quote.test.js#L185-L186).

However, as discovered in [this comment](https://github.com/WordPress/gutenberg/issues/46519#issuecomment-1351644708), the arrow-up key may cause unstable behavior depending on the theme, so it is rewritten as the arrow-left key.

Reply:
Maybe we should leave an inline comment here. What do you think?